db/schema.rb

```diff
@@ -199,8 +199,6 @@
   create_table "groups", force: :cascade do |t|
     t.string "name"
     t.text "description"
-    t.string "phone_number"
```

Comment:
@skng5 Can you please make this go away?

Reply:
Fixed the schema.db and fixed my process.